project/BuildSettings.scala

```diff
@@ -16,53 +16,57 @@ import sbt._
 object BuildSettings extends AutoPlugin {
   override def trigger: PluginTrigger = allRequirements
 
-  override def projectSettings: Seq[Def.Setting[_]] = Seq(
-    organization := "org.hatdex",
-    version := "2.6.9",
-    resolvers ++= Dependencies.resolvers,
-    scalaVersion := Dependencies.Versions.scalaVersion,
-    crossScalaVersions := Dependencies.Versions.crossScala,
-    name := "HAT Client Scala Play",
-    description := "HAT HTTP API wrapper in Scala",
-    licenses += ("Mozilla Public License 2.0", url("https://www.mozilla.org/en-US/MPL/2.0")),
-    scmInfo := Some(
-      ScmInfo(
-        url("https://github.com/Hub-of-all-Things/hat-client-scala-play"),
-        "scm:git@github.com:Hub-of-all-Things/hat-client-scala-play.git"
-      )
-    ),
-    homepage := Some(url("https://hubofallthings.com")),
-    developers := List(
-      Developer(
-        id    = "AndriusA",
-        name  = "Andrius Aucinas",
-        email = "andrius@smart-e.org",
-        url   = url("http://smart-e.org")
-      )
-    ),
-    scalacOptions ++= Seq(
-      "-deprecation", // Emit warning and location for usages of deprecated APIs.
-      "-feature", // Emit warning and location for usages of features that should be imported explicitly.
-      "-unchecked", // Enable additional warnings where generated code depends on assumptions.
-      "-Xfatal-warnings", // Fail the compilation if there are any warnings.
-      "-Xlint", // Enable recommended additional warnings.
-      "-Ywarn-adapted-args", // Warn if an argument list is modified to match the receiver.
-      "-Ywarn-dead-code", // Warn when dead code is identified.
-      "-Ywarn-inaccessible", // Warn about inaccessible types in method signatures.
-      "-Ywarn-nullary-override", // Warn when non-nullary overrides nullary, e.g. def foo() over def foo.
-      "-language:postfixOps", // Allow postfix operators
-      "-Ywarn-numeric-widen" // Warn when numerics are widened.
+  override def projectSettings: Seq[Def.Setting[_]] =
+    Seq(
+      organization := "org.hatdex",
+      version := "2.6.10-SNAPSHOT",
```

Comment:
Please review versioning changes suggested by @benshaw in PR https://github.com/Hub-of-all-Things/hat-client-scala-play/pull/20